Fix help text about standard input usage

davidBar-On opened the pull request:
Suggested fix for issue #14259 - change help text to require using "-" for input from the standard-input.

davidBar-On committed:
Fix issue 14259 - help text about standard input usage

nikola-matic reviewed (commented):
Hi @davidBar-On, can you please rebase on top of the latest develop in order to fix the failing CI steps?

davidBar-On commented:
Hi @nikola-matic, when running `git rebase upstream/develop` I get the message "Current branch develop is up to date" (upstream is " https://github.com/ethereum/solidity.git").  This PR code branch is based on the develop branch, so it seems that is up to date.  Am I doing something wrong?

nikola-matic commented:
> Hi @nikola-matic, when running `git rebase upstream/develop` I get the message "Current branch develop is up to date" (upstream is " https://github.com/ethereum/solidity.git"). This PR code branch is based on the develop branch, so it seems that is up to date. Am I doing something wrong?

Did you update `upstream/develop`?

davidBar-On commented:
Oops! My mistake.  I forgot that I updated the develop branch after creating the PR's branch ... Now rebased.

However, several tests are still failing because of authorization error: "curl: (22) The requested URL returned error: 401".  I am not sure whether or not this is related to my Github environment.

nikola-matic commented:
> Oops! My mistake. I forgot that I updated the develop branch after creating the PR's branch ... Now rebased.
> 
> However, several tests are still failing because of authorization error: "curl: (22) The requested URL returned error: 401". I am not sure whether or not this is related to my Github environment.

Yup, we reworked some of our CI, which then causes issues for external contributors since you end up hitting the rate limit for the Github API, hence the 401s. We've merged a fix for this last week, which should hopefully resolve this (although there's still a possibility that the rate limit will be hit again). I've checked out your fork, rebased against `ethereum:develop`, so hopefully it should go through now.

For future reference, when working with forked repositories such as as in this case, you first wanna add the upstream remote to your repo, fetch it, then rebase from it, as below:

```
git remote add upstream git@github.com:ethereum/solidity.git
git fetch upstream
# assuming you're on your local branch
git rebase upstream/develop
```

davidBar-On commented:
@nikola-matic, thanks a lot for your input.  I will make sure to follow your instructions in the future.
